pkgs/development/compilers/edk2/default.nix

```diff
@@ -61,6 +61,7 @@ edk2 = stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pythonEnv ];
   depsBuildBuild = [ buildPackages.stdenv.cc buildPackages.util-linux buildPackages.bash ];
+  buildInputs = [ libuuid ];
```

Comment:
Weird, `libuuid` used to be here but was removed in https://github.com/nixos/nixpkgs/commit/9ccccc3e255948be13079138d09b3b21da1a5e23 "edk2: fix cross compilation" @symphorien 

`pkgsCross.aarch64-multiplatform.edk2` builds on master??

Reply:
libuuid and util-linux are aliases of one-another.

Reply:
But strictDeps is enabled, it's probably the depsBuildBuild compiler finding it 

Why is a build build compilers needed here